### ISSUE TITLE
Bug: v1.1.0 fails to generate BOM on projects with no dependencies

### ISSUE THREAD

**Issue by aarongoldenthal** [CycloneDX/cyclonedx-node-module#36]
Starting with version 1.1.0, the following error is encountered when running on a project with no dependencies:

```
$ npm install -g @cyclonedx/bom
/usr/local/bin/cyclonedx-bom -> /usr/local/lib/node_modules/@cyclonedx/bom/bin/cyclonedx-bom
+ @cyclonedx/bom@1.1.0
added 41 packages from 37 contributors in 2.149s
$ cyclonedx-bom -o bom.xml
/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/xmlbuilder/lib/XMLNode.js:191
          throw new Error("Could not create any elements with: " + name + ". " + this.debugInfo());
          ^
Error: Could not create any elements with: . node: <components>, parent: <bom>
    at XMLElement.element (/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/xmlbuilder/lib/XMLNode.js:191:17)
    at XMLElement.ele (/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/xmlbuilder/lib/XMLNode.js:611:21)
    at /usr/local/lib/node_modules/@cyclonedx/bom/index.js:208:27
    at /usr/local/lib/node_modules/@cyclonedx/bom/node_modules/read-installed/read-installed.js:142:5
    at /usr/local/lib/node_modules/@cyclonedx/bom/node_modules/read-installed/read-installed.js:263:14
    at cb (/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/slide/lib/async-map.js:47:24)
    at /usr/local/lib/node_modules/@cyclonedx/bom/node_modules/read-installed/read-installed.js:263:14
    at cb (/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/slide/lib/async-map.js:47:24)
    at /usr/local/lib/node_modules/@cyclonedx/bom/node_modules/read-installed/read-installed.js:263:14
     at cb (/usr/local/lib/node_modules/@cyclonedx/bom/node_modules/slide/lib/async-map.js:47:24)
ERROR: Job failed: exit code 1
```

The output does appear correct if `devDependencies` are included.

Previous versions would generate the following output:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<bom xmlns="http://cyclonedx.org/schema/bom/1.1" version="1" serialNumber="urn:uuid:3b14e096-3de2-4fec-81ce-7ad082ff99fe">
    <components>
    </components>
</bom>
```

**Comment by stevespringett** [comment]
Thanks for the report

**Comment by stevespringett** [comment]
Fixed in v1.1.1.
